.github/workflows/testing.yml

```diff
@@ -18,18 +18,22 @@ jobs:
     steps:
     - uses: actions/checkout@v2
         
-    - name: Update dependencies if changed
+    - name: Reinstall dependencies if changed
       run: |
-        cmp -s /meta.yaml localbuild/meta.yaml || (echo Dependencies differ \
+        cmp -s /meta.yaml localbuild/meta.yaml && cmp -s /development.txt requirements.d/development.txt || (echo Dependencies differ \
         && cat localbuild/meta.yaml \
         | sed -n '/^requirements:/,/^test:/p' \
         | sed -e "s/.*- //" \
         | sed -e "s/menuinst.*//" \
         | sed -e "s/.*://" > reqs.txt \
+        && source /opt/conda/bin/activate mssenv \
```

Comment:
I've no idea if it makes a difference to have first the env activated or the channels updated

I think that it has to be done before we install something, which in both cases is done